crates/spfs/src/runtime/spec_api_version.rs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Contributors to the SPK project.
+// SPDX-License-Identifier: Apache-2.0
+// https://github.com/spkenv/spk
+
+use serde::{Deserialize, Serialize};
+
+/// Supported spfs spec api versions
+#[derive(Debug, Deserialize, Serialize, Copy, Clone, Eq, PartialEq, strum::Display)]
+pub enum SpecApiVersion {
+    #[serde(rename = "v0/layer", alias = "v0/livelayer")]
+    V0Layer,
+    #[serde(rename = "v0/layerlist")]
```

Comment:
my final hesitation with `layerlist` is that it feels too explicit - eg we won't be able to extend the data in this file without the name becoming confusing. Did we already consider `v0/platform`? Because I could also see this being a really useful way to define and then commit platforms directly

Reply:
It is closer to an EnvSpec. It's basically anEnvSpec in a file. How about `v0/envspec`? 
Or `v0/referencelist` to line up with what `spfs run -h` calls an env spec (`REFERENCE`).

Reply:
`v0/runspec`!